docs/postgresql/topics/client_auth.md

```diff
@@ -0,0 +1,40 @@
+Move your `{{ ca_cert }}` certificate to your PostgreSQL data directory&mdash;often at `/var/lib/pgsql/data` or `/usr/local/pgsql/data`&mdash;and name it `root.crt` (the usual convention, though other paths are possible).
+
+```shell-session
+$ sudo cp {{ ca_cert }} /var/lib/pgsql/data/root.crt
+```
+
+You'll need to maually set the file permissions to disallow access to world or group.
+
+```shell-session
+$ sudo chown postgres:postgres /var/lib/pgsql/data/root.crt
+$ sudo chmod 0600 /var/lib/pgsql/data/root.crt
+```
```

Comment:
A certificate is not something super sensitive, only the public key is there.

Reply:
Hmm, good point. I know postgres requires `600` for keys, but not 100% sure if it does for certs. I can do some testing and remove this section if it doesn't care. :+1:

Reply:
It's not critical at all. It's ok to use 600.

Reply:
Tested without 0600 on non-key files, and all good! Removed that part from docs.